habitat_sim/utils/datasets_download.py

```diff
@@ -505,21 +505,21 @@ def main(args):
     args = parser.parse_args(args)
     replace = args.replace
 
-    # get a default data_path from git
+    # get a default data_path "./data/"
     data_path = args.data_path
     if not data_path:
         try:
-            import git
```

Comment:
Maybe, we can do both, check git repo first, and then use `./data/` as fall back?